README.md

```diff
@@ -46,6 +60,33 @@ apps:
       openAIKey: $OPENAI_API_KEY
 ```
 
+### Using Azure OpenAI
+
+To provision the plugin to use Azure OpenAI, use settings similar to this:
+
+```yaml
+apiVersion: 1
+
+apps:
+  - type: 'grafana-llm-app'
+    disabled: false
+    jsonData:
+      openAI:
+        provider: azure
+        url: https://<resource>.openai.azure.com
+        azureModelMapping:
+          - ["gpt-3.5-turbo", "gpt-35-turbo"]
+    secureJsonData:
+      openAIKey: $OPENAI_API_KEY
+```
+
+where:
+
+- `<resource>` is your Azure OpenAI resource name
+- the `azureModelMapping` field contains `[model, deployment]` pairs so that features know
+  which Azure deployment to use in place of each model you wish to be used.
+
+
 ## Adding LLM features to your plugin or Grafana core
 
 To make use of this plugin when adding LLM-based features, you can use the helper functions in the `@grafana/experimental` package.
```

Comment:
The line below here still says "correct version", could we use "latest" version as in https://github.com/grafana/grafana-llm-app/blob/4b941a67aeb4ce438bf62f4c10a27b8e7c8be72e/src/README.md#for-plugin-developers?

Reply:
may just need to refresh the page: 
![image](https://github.com/grafana/grafana-llm-app/assets/350728/4e316b5e-d708-47d5-8d87-ffc63f858232)